azure-devops/projects/io-backend-projects/service_connections.tf

```
@@ -85,5 +85,20 @@ resource "azuredevops_serviceendpoint_npm" "pagopa-npm-bot" {
   project_id            = azuredevops_project.project.id
   service_endpoint_name = "pagopa-npm-bot"
   url                   = "https://registry.npmjs.org"
-  access_token          = data.azurerm_key_vault_secret.key_vault_secret["pagopa-npm-bot-TOKEN"].value
+  access_token          = module.secrets.values["pagopa-npm-bot-TOKEN"].value
+}
+
+module "PROD-IO-TLS-CERT-SERVICE-CONN" {
+  depends_on = [azuredevops_project.project]
+  source     = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_serviceendpoint_azurerm_limited?ref=v1.1.0"
+
+  project_id        = azuredevops_project.project.id
+  name              = "prod-io-tls-cert"
```

```suggestion
  name              = "io-p-tls-cert"
```
should be `io-p-tls-cert` from https://github.com/pagopa/gitops/blob/main/azure-devops/projects/pagopa-projects/service_connections.tf#L104